nightingale_visualization/package.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
```

Comment:
Not sure why these changes are here. Might have to move my changes to a different branch to avoid this

Reply:
kk just lmk when ur ready for a review

Reply:
i prob shouldve merged in stuff from my branch a while back. it pretty much has sim+arm+nav+misc now which is a mess. i might try to split things up soon

Reply:
I think it's ready for review thanks! I remember you mentioned to go onto the nav-bringup branch to run sim which is why I branched off it so I can test the point cloud stuff in sim. Do I need to be on the nav-bringup branch to run sim or will any branch work? If I don't need to be off sim I'll move my stuff to avoid the conflicts to make things easier. I was able to update my local nav-bringup branch to get rid of most of the file differences but I still see one in the nav-sim folder. Is the nav-bringup branch branched off of a different branch? I'm trying to figure out where things are atm and where I should by my stuff